docs/segment.md

```diff
@@ -2,98 +2,188 @@
+Segmentation can result from a prediction, can be performed manually, or can be imported with the **import** button when you import an XML file directly from the **Images** tab inside a document.
 
-Therefore, this section mainly introduces the **segmentation panels** and some related features. The segmentation panel is one of the panels available in the "Edit" page, inside a document's dashboard (`{base_url}/document/{document-id}/images/`), which is available:
+!!! warning
+    Please make sure to segment your document before transcribing, otherwise, you will risk erasing your transcribed data.
 
-- by clicking on the "Edit" button once you are in your document dashboard (it will send you to the last edited page of the document),
-- or by clicking on the "Edit" button of a page thumbnail, in the Images tab.
+## Text regions segmentation
 
-The segmentation panel is toggled by clicking on the "Segmentation" button at the top of the page (or by pressing ++control+3++).
+In the following example, we segmented and labeled the zones that bear a semantic role for our comprehension of the text. We are dealing with a poem, so we drew with eScriptorium bounding boxes around the page number (in blue), the title (in purple), and the stanzas (in green).
+
+However, you could have decided to annotate the layout more precisely, for example, all the verses inside the stanzas. It is up to you to decide how deep you want to go into the text regions segmentation, depending on your needs.
+
+![image: Illustration of several segmented text regions](img/segment/text_regions_segmentation.png "Several segmented text regions are visualized with eScriptorium interface")
+
+### Segmenting text regions with eScriptorium 
+
+You can draw manually draw bounding boxes inside the Segmentation panel. To do so, switch from baseline editing mode to region mode, by clicking the **mosaic** button, or by pressing ++r++.
+
+You can now draw bounding boxes around the text regions you want to annotate by clicking and dragging a zone (green by default).
+
+![image: Drawing bounding boxes in the Segmenting panel of eScriptorium](img/segment/bounding_boxes_drawing.gif "Several bounding boxes are drawn around meaningful text regions.")
+
+You can modify a text region by clicking on it, and by dragging it to the dimensions you want.
+
+![image: Resizing a text region](img/segment/resizing_text_region.gif "")
+
+You can delete a text region by clicking on it, still in region mode, and by clicking on the delete button, or by pressing ++delete++.
+
+!!! note
+    Keep in mind that you can always press ++ctrl+z++ to undo any modification.
+
+### Labeling segmented text regions with an ontology
+
+eScriptorium allows to label segmented text regions with an ontology that you can customize.
```

Comment:
Shouldn't we do an entry for ontology on the terminology or at least make a referral to what is an ontology ?

Reply:
Good point, I think it would be best to address it in the terminology page